docs/features.mdx

```diff
@@ -15,15 +15,17 @@ import { TopBanners } from "@site/src/components/TopBanners";
 
 - 📱 **Responsive Design**: Enjoy a seamless experience across Desktop PC, Laptop, and Mobile devices.
 
-- 📱 **Progressive Web App (PWA) for Mobile**: Enjoy a native app-like experience on your mobile device with our PWA, providing offline access on localhost and a seamless user interface.
+- 📱 **Progressive Web App (PWA) for Mobile**: Enjoy a native progressive web app experience on your mobile device, featuring offline access on localhost or a personal domain, and a smooth user interface.
+To set up a PWA, you'll need some understanding of technologies like Linux, Docker, and reverse proxies such as `Nginx`, `Caddy`, or `Traefik`. Using these tools can help streamline the process for creating and deploying your PWA tailored to your needs.
```

Comment:
I believe PWA doesn't require https? I might be wrong here

Reply:
> I believe PWA doesn't require https? I might be wrong here

It does sadly.

Reply:
> I believe PWA doesn't require https? I might be wrong here

For a web app to be installable, it must be served in a [secure context](https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps). This usually means that it must be served over HTTPS. Local resources, such as `localhost`, `127.0.0.1` and `file://` are also considered secure.